src/lib/config.ts

```diff
@@ -525,16 +536,20 @@ export class default_config {
     }
 
     /**
-     * Whether to use the keytranslatemap in various maps.
+     * Whether to use the keytranslatemap
```

Comment:
```suggestion
     * Whether to use the keytranslatemap. For example, useful for changing the effective base layout when using [[alwaysqwerty]].
     * 
     * To make your `alwaysqwerty` layout more like a GB layout, run `:keymap @ "` and then no matter which layout you are using, pressing the `"` key on a physical GB layout will trigger `"` binds rather than `@` binds.
     * 
     * As a rule of thumb: `:keymap [equivalent US key] [key you want to press on your physical layout]`
```

I finally understand the point of using `keymap` and `alwaysqwerty` at the same time now! This help message isn't perfect but I think it's at least better :)